browser-extension/utils.js

```diff
@@ -16,6 +16,16 @@ export const alertUseOnLinkToYoutube = () => {
   alertOnCurrentTab('This must be used on a link to a youtube video')
 }
 
+export const alertInvalidAccessToken = () => {
+  alertOnCurrentTab(
+    'It seems you are logged in Tournesol, but your connection needs to be refreshed.\\n\\n' +
```

Comment:
phrasing could be: "...logged in on Tournesol" ?

Reply:
sure!